pootle/templates/user/profile.html

```diff
@@ -18,8 +18,8 @@ <h1 class="user-name">
         {{ object.display_name }}
       {% endif %}
     </h1>
-    {% if user.is_superuser or request.permissions.administrate %}
-    <p><a href="mailto:{{ object.email }}">{{ object.email }}</a></p>
+    {% if user.is_superuser or user.is_manager %}
```

Comment:
Caching it has the downside that we have to get rid of the cache every time the permissions are changed. I am ok with setting this on the context provided by the view.